save-cloud-common/src/commonMain/kotlin/org/cqfn/save/repository/GitRepository.kt

```diff
@@ -13,7 +13,7 @@ import kotlinx.serialization.Serializable
  */
 @Serializable
 data class GitRepository(
-    val url: String,
+    val url: String?,
```

Comment:
Why is this nullable now?

Reply:
Because user may not provide a project link and upload project only as binary file.

Reply:
Ok, then url should be nullable in Project. Why is `GitRepository` affected?